sencha/index.py

```diff
@@ -154,11 +166,20 @@ def maybe_make_peptide_bloom_filter(
                 )
     else:
         peptide_ksize = get_peptide_ksize(molecule, peptide_ksize)
-        logger.info(
-            f"Creating peptide bloom filter with file: {peptides}\n"
-            f"Using ksize: {peptide_ksize} and alphabet: {molecule} "
-            f"..."
-        )
+        if len(peptides) == 1:
+            logger.info(
+                f"Creating peptide bloom filter with file: {peptides[0]}\n"
+                f"Using ksize: {peptide_ksize} and alphabet: {molecule} "
+                f"..."
+            )
+        else:
+            index_dir = os.path.dirname(peptides[0])
```

Comment:
in line 151 you have already set peptides = peptides[0] why would we have to index that?

Reply:
line 151 only sets peptides = peptides[0] for the `peptides_are_bloom_filter` case. If instead we need to build the bloom filter (line 167-189), we still need the full peptides list